File: ga/4omini_result_2024_7/result/var_func_mini/18_var_func.cpp
fn print_values(values: &[i32]) {
    for &value in values {
        println!("Value: {}", value);
    }
}

fn main() {
    let values = vec![10, 20, 30];  // creating a Vec to hold the values
    print_values(&values);           // passing a reference to the slice of values
}

File: ga/gpt3_5_result_2024_6/result/var_func_gpt/18_var_func.rs
use std::ffi::VaList;

fn print_values(n: i32, args: VaList) {
    for _ in 0..n {
        let value = args.arg::<i32>();
        println!("Value: {}", value);
    }
}

fn main() {
    let args = VaList::from([10, 20, 30]);
    print_values(3, args);
}